tests/inference/models_predictions_tests/conftest.py

```diff
@@ -146,6 +146,17 @@ def yolonas_det_model() -> Generator[str, None, None]:
     shutil.rmtree(model_cache_dir)
 
 
+@pytest.fixture(scope="function")
+def yolov10_det_model() -> Generator[str, None, None]:
+    model_id = "yolov10_det/1"
+    model_cache_dir = fetch_and_place_model_in_cache(
+        model_id=model_id,
+        model_package_url="https://storage.googleapis.com/roboflow-tests-assets/yolov10_det.zip",
```

Comment:
Wondering if there is another step here other than uploading this zip to google cloud.

Reply:
there should be adjustment of access policy required

Reply:
ok, was wrong - problem was rather in content of ZIP